components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx

```diff
@@ -1 +1,43 @@
-export const HowItWorks = () => <h1>How it works</h1>;
+import { urlForImage } from '@/lib/sanity';
+import Image from 'next/image';
+import { ImagesListItem } from '@/types';
+
+type Props = {
+  howItWorksImages: ImagesListItem[];
+  howItWorksBlock: string;
```

Comment:
Use shorter components names:
```suggestion
  images: ImagesListItem[];
  title: string;
```

Reply:
Did changes on DE side, still do not good tool to rename sanity schema